plutus-playground-client/src/Editor/State.purs

```diff
@@ -70,6 +75,22 @@ handleAction bufferLocalStorageKey (HandleDropEvent event) = do
   void $ query _editorSlot unit $ tell $ Monaco.SetPosition { column: 1, lineNumber: 1 }
   saveBuffer bufferLocalStorageKey contents
 
+handleAction _ (SetFeedbackPaneDragStart event) = do
+  liftEffect $ preventDefault event
+  assign _feedbackPaneDragStart $ Just $ pageY event
+
+handleAction _ ClearFeedbackPaneDragStart = do
+  feedbackPaneExtend <- use _feedbackPaneExtend
+  assign _feedbackPaneDragStart Nothing
+  assign _feedbackPanePreviousExtend feedbackPaneExtend
+
+handleAction _ (FixFeedbackPaneExtend mouseY) = do
+  feedbackPaneDragStart <- use _feedbackPaneDragStart
+  feedbackPanePreviousExtend <- use _feedbackPanePreviousExtend
+  case feedbackPaneDragStart of
+    Nothing -> pure unit
+    Just startMouseY -> assign _feedbackPaneExtend $ clamp 0 100 (startMouseY - mouseY + feedbackPanePreviousExtend)
```

Comment:
One alternative to this pattern match is:

``` purescript
for_ feedbackPaneDragStart 
  (\startMouseY ->
    assign _feedbackPaneExtend
      $ clamp 0 100 
      $ startMouseY - mouseY + feedbackPanePreviousExtend)
```

Reply:
Nice, thanks.